corehq/apps/app_manager/suite_xml/generator.py

```diff
@@ -45,6 +49,11 @@ def generate_suite(self):
             DetailContributor(self.suite, self.app, self.modules, self.build_profile_id),
         ])
 
+        if self.app.supports_practice_users and self.app.get_practice_user(self.build_profile_id):
```

Comment:
should this be `app.enable_practice_user` i.e. should it include the domain privilege check?